src/common/Footer.jsx

```diff
@@ -5,8 +5,10 @@ import logo from '../images/ffiec-logo.svg'
 import { getKeycloak } from '../utils/keycloak.js'
 
 export const getLink = () => {
-  if (getKeycloak().authenticated) return '/filing/2018/institutions'
-  return '/filing/2018/'
+  var year = window.location.pathname.substring(8,12)
```

Comment:
I think instead of grabbing the year out of the pathname via `substring`, we should just pass the the `year` down to the component or access it from the redux store.